src/Components/test/E2ETest/Tests/MonoSanityTest.cs

```diff
@@ -22,6 +22,7 @@ public class MonoSanityTest : ServerTestBase<AspNetSiteServerFixture>
             : base(browserFixture, serverFixture, output)
         {
             serverFixture.BuildWebHostMethod = MonoSanity.Program.BuildWebHost;
+            serverFixture.Environment = AspNetEnvironment.Development;
```

Comment:
?

Reply:
This is needed because it is using static web assets now and the runtime piece is only enabled in development

Reply:
Just to clarify: in general does this mean we have to warn developers only to run in `Development` environment now, and that `Production` environment will only work for published applications?

Reply:
Yes, that's what we decided for 3.0 with SWA. We don't want it to be turned on in production by default, but if someone wants it they can turn it on explicitly by calling `webHost.UseStaticWebAssets()` unconditionally.

The goal here was that in Production (published app) you wouldn't need it and we wouldn't want it there, so its only turned on by default in development, and you get to turn it on in other environments if you want to do so.